src/components/MAPI/keypairs/ClusterDetailKeyPairs.tsx

```diff
@@ -80,17 +82,30 @@ const ClusterDetailKeyPairs: React.FC<IClusterDetailKeyPairsProps> = () => {
   const { clusterId, orgId } =
     useParams<{ clusterId: string; orgId: string }>();
 
+  const organizations = useSelector(selectOrganizations());
+  const selectedOrg = orgId ? organizations[orgId] : undefined;
+  const namespace = selectedOrg?.namespace;
+
+  const provider = window.config.info.general.provider;
+
```

Comment:
Nothing new here - this is just moved from further down the file so we can access the `namespace` and `provider` variables